EBH-5123/CSV_merge.py

```
@@ -0,0 +1,79 @@
+        output_filename = "Merged_WGS_CSV_data.csv"
+        output_path = os.path.join(output_path, output_filename)
+        merged_df.to_csv(output_path, index=False)
+        
+        print("\n--- Success! ---")
+        print(f"Successfully processed and merged {len(all_dfs)} files.")
+        print(f"Master file saved at: {output_path}")
+    else:
+        print("No valid data was merged.")
+
+if __name__ == "__main__":
+    merge_genetics_csv(BASE_PATH, OUT_PATH)
```

Ruff (0.15.15)

[error] Undefined name `OUT_PATH`

(F821)